bin/cml-runner.js

```diff
@@ -315,45 +315,48 @@ const opts = decamelize(
   yargs
     .usage(`Usage: $0`)
     .default('labels', RUNNER_LABELS)
-    .describe('labels', 'Comma delimited runner labels')
+    .describe(
+      'labels',
+      'One or more user-defined labels for this runner (delimited with commas)'
+    )
     .default('idle-timeout', RUNNER_IDLE_TIMEOUT)
     .describe(
       'idle-timeout',
-      'Time in seconds for the runner to be waiting for jobs before shutting down. 0 waits forever.'
+      'Time in seconds for the runner to be waiting for jobs before shutting down. Setting it to 0 disables automatic shutdown'
     )
     .default('name', RUNNER_NAME)
-    .describe('name', 'Name displayed in the repo once registered')
+    .describe('name', 'Name displayed in the repository once registered')
 
     .boolean('single')
     .default('single', RUNNER_SINGLE)
-    .describe('single', 'If specified, exit after running a single job.')
+    .describe('single', 'Exit after running a single job')
     .boolean('reuse')
     .default('reuse', RUNNER_REUSE)
     .describe(
       'reuse',
-      "If specified, don't spawn a new runner if there is a registed runner with the given labels."
+      "Don't launch a new runner if an existing one has the same name or overlapping labels"
     )
 
     .default('driver', RUNNER_DRIVER)
-    .describe('driver', 'If not specify it infers it from the ENV.')
+    .describe('driver', 'Platform where the repository is hosted')
     .choices('driver', ['github', 'gitlab'])
     .default('repo', RUNNER_REPO)
     .describe(
       'repo',
-      'Specifies the repo to be used. If not specified is extracted from the CI ENV.'
+      'Repository to be used for registering the runner. If not specified, it will be inferred from the environment.'
```

Comment:
```suggestion
      'Repository to be used for registering the runner. If not specified, it will be inferred from the environment'
```

Remove stray period